src/index.ts

```diff
@@ -55,6 +55,39 @@ export interface CreateZoneRequest {
   dnsName: string;
   description?: string;
   name?: string;
+  dnssecConfig?: ManagedZoneDnsSecConfig;
```

Comment:
The API already excepted this field, but we weren't setting it?

Reply:
Yeah, there are _many_ fields this thing accepts that we're not currently exposing.  For now, this one gets us out of overground jail though.